### DISCUSSION TITLE
unable to delete topics, if topic name is beginning with underscore (_)

### DISCUSSION THREAD

**Discussion by nitin1677** [provectus/kafka-ui#2755] [Q&A] [answered]
Kafka UI Version: d149d26013a81e22e5d0c8b59db56ac0565b9e55 ( [d149d26](https://github.com/provectus/kafka-ui/commit/d149d26))

Remove topic options are disabled for topics, where topic-name starts with underscore (_). what are the options to enable delete for topics, where name starts with underscore (_).

**Comment by Haarolean** [comment]
Hey, I'll convert this to an issue

**Comment by Haarolean** [comment, marked as answer]
Solved within #2759
